views/static/home.handlebars

```diff
@@ -2,21 +2,32 @@
   <div class="container text-center">
     <div class="row">
       <div class="my-5 col-12 col-md-6">
-        <h1 class="text-blue">Matching Donors with Needy people in the Society</h1>
+        <h1 class="text-success">Helping each other has never been more convenient</h1>
         <div class="my-5 row text-center">
           <div class="col-12 col-md-6 text-center">
             <h2>For Donors</h2>
-            <p class="text-muted">Donation can be made easy by simply uploading an item to the page. Eveyone can donate
-              or
-              become a donor. </p>
-            <a href="/auth/signup?uType=Donor" class="btn btn-outline-secondary">Start Donating</a>
+            <p class="text-muted">
+              We are here to ensure you can help or ask for help,
+              in the most convenient way possible. Helping others
+              is always an emotionally rewarding experience, and
+              now we aim to make it convenient too.
+              <br>
+              Join Us, and help make the world a better place:
+            </p>
+            <a href="/auth/signup" class="btn btn-outline-secondary">Sign up to Start Helping</a>
```

Comment:
Please rebase your branch with master @neilvarunnaidu 

cc- @SonaliChavan95

Reply:
`/auth/signup?uType=Donor` for donor signup and `/auth/signup?uType=Recipient` for recipient sign up.

Reply:
Please make this fix wherever you are using signup even on other pages. @neilvarunnaidu